frontend/public/components/storage-class-form.tsx

```diff
@@ -862,10 +862,12 @@ export class StorageClassForm_ extends React.Component<
       if (parameter.Component) {
         const { Component } = parameter;
         return (
-          <Component
-            key={key}
-            onParamChange={(value: string) => this.setParameterHandler(key, value, false)}
-          />
+          <div className="form-group">
```

Comment:
Storage-class-form is using form-group for static content and for dynamic content expecting us to use this tag.

The problem is ocs-sc-from is not using form-group for the encryption checkbox. Since we are using callback in between  other dynamic content in extension, this problem will keep occurring. It is better we call dynamic content between form-group tag


I checked for other places, for now only we are using this callback feature, so we can take this change.